src/libraries/System.Reflection.Metadata/tests/Metadata/Ecma335/Encoding/ControlFlowBuilderTests.cs

```diff
@@ -419,6 +419,50 @@ public void Branch_LongInstruction_LongDistance()
             }, builder.ToArray());
         }
 
+        [Fact]
+        public void Switch()
```

Comment:
FYI: the test failing on CI

Might be interesting to add a test that emits switch and try catch both

Reply:
I fixed the tests and CI is now green.

I don't see much value in such a test, switches and exception handlers are orthogonal to each other.